frontend/src/metabase/admin/settings/components/widgets/PersistedModelRefreshIntervalWidget.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import Radio from "metabase/core/components/Radio";
+
+import { PersistedModelsApi } from "metabase/services";
+
+type Props = {
+  setting: {
+    key: string;
+    value: number;
+    default: number;
+    options: Record<string, string>;
+  };
+  disabled?: boolean;
+  onChangeSetting: (setting: string, value: unknown) => void;
+};
+
+const PersistedModelRefreshIntervalWidget = ({
+  setting,
+  disabled,
+  onChangeSetting,
+}: Props) => {
+  return (
+    <Radio
+      className="mt2"
+      variant="bubble"
+      disabled={disabled}
+      value={setting.value || setting.default}
+      onChange={async hours => {
```

Comment:
Could we remove this widget and use "radio" field type directly? There is `onChanged`https://github.com/metabase/metabase/blob/75d23718428d8efb6bb8a64bb677cca1d511a7de/frontend/src/metabase/admin/settings/selectors.js#L91 handler you could use to call the API